responsive-css.md

```diff
@@ -64,52 +64,7 @@ One of the most popular css frameworks to use is Bootstrap. Bootstrap combines h
 
 It can be included via CDN and can be integrated into nearly any part of your webpage.
 
-```html
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <!-- Required meta tags -->
-    <meta charset="utf-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, shrink-to-fit=no"
-    />
-
-    <!-- Bootstrap CSS -->
-    <link
-      rel="stylesheet"
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
-      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
-      crossorigin="anonymous"
-    />
-
-    <!-- Optional JavaScript -->
-    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
-    <script
-      src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
-      integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
-      crossorigin="anonymous"
-    ></script>
-    <script
-      src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"
-      integrity="sha384-UO2eT0CpHqdSJQ6hJty5KVphtPhzWj9WO1clHTMGa3JDZwrnQq4sF86dIHNDz0W1"
-      crossorigin="anonymous"
-    ></script>
-    <script
-      src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
-      integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
-      crossorigin="anonymous"
-    ></script>
-
-    <title>Bootstrap Example Page</title>
-  </head>
-  <body>
-    <div class="jumbotron">
-      <h1>My First Bootstrap Page</h1>
-    </div>
-  </body>
-</html>
-```
+If you'd like to see a working Bootstrap example, click [here](https://codesandbox.io/s/vq430zj5n3)
```

Comment:
I think we should stick to using the examples from the Bootstrap documentation, as they show more than just a page header. I also wonder if the codesandbox site might be a bit overwhelming for someone that we are assuming is brand new to html and css.

https://getbootstrap.com/docs/4.3/examples/

Reply:
Doesn’t seem overwhelming to me, imo. Linking to the docs should be sufficient enough though

Reply:
Also consider that the first thing you see when clicking on the link is a `package.json`